airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -272,11 +272,12 @@ def read_records(
 
             if job_status in ["JobComplete", "Aborted", "Failed"]:
                 self.delete_job(url=job_full_url)
-                pagination_complete = True
+                if job_status in ["Aborted", "Failed"]:
```

Comment:
should we be raising exception if the job failed?

Reply:
Done